src/components/RepoItem/RepoItem.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import styles from "../RepoItem/RepoItem.module.css";
+import classnames from "classnames";
+
+const RepoItem = ({
+  url,
+  name,
+  language,
+  stargazers_count,
+  forks_count,
+  updated_at,
+}) => {
+  return (
+    <div>
+      <a
+        href={url}
+        className={styles.repo_name_link}
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        {name}
+      </a>
+      <div className={styles.info_about_repo}>
+        <span
+          className={classnames({
+            [styles.language]: true,
+            [styles.html_circle]: language === "HTML",
+            [styles.css_circle]: language === "CSS",
+            [styles.js_circle]: language === "JavaScript",
+          })}
+        >
+          {language}
+        </span>
+        <span className={styles.star}>{stargazers_count}</span>
```

Comment:
Codacy found an issue: [Identifier 'stargazers_count' is not in camel case.](https://app.codacy.com/gh/WebHeroSchool/tododo/pullRequest?prid=5504860)